hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/AbiConstants.java

```diff
@@ -1,119 +1,133 @@
-	//ownerOf(uint256 tokenId)
-	public static final int ABI_ID_ERC_OWNER_OF_NFT = 0x6352211e;
-	//tokenURI(uint256 tokenId)
-	public static final int ABI_ID_ERC_TOKEN_URI_NFT = 0xc87b56dd;
-	//Transfer(address indexed from, address indexed to, uint256 indexed tokenId)
-	//Transfer(address indexed from, address indexed to, uint256 value)
-	public static final Bytes TRANSFER_EVENT = Bytes.fromHexString(
-			"ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef");
-	//Approval(address indexed owner, address indexed spender, uint256 value)
-	//Approval(address indexed owner, address indexed approved, uint256 indexed tokenId)
-	public static final Bytes APPROVAL_EVENT = Bytes.fromHexString(
-			"8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925");
-	//ApprovalForAll(address indexed owner, address indexed operator, bool approved)
-	public static final Bytes APPROVAL_FOR_ALL_EVENT = Bytes.fromHexString(
-			"17307eab39ab6107e8899845ad3d59bd9653f200f220920489ca2b5937696c31");
+    // **** HIP-218 + HIP-376 function selectors and event signatures ****
+    // redirectForToken(address token, bytes memory data)
+    public static final int ABI_ID_REDIRECT_FOR_TOKEN = 0x618dc65e;
+    // name()
+    public static final int ABI_ID_ERC_NAME = 0x06fdde03;
+    // symbol()
+    public static final int ABI_ID_ERC_SYMBOL = 0x95d89b41;
+    // decimals()
+    public static final int ABI_ID_ERC_DECIMALS = 0x313ce567;
+    // totalSupply()
+    public static final int ABI_ID_ERC_TOTAL_SUPPLY_TOKEN = 0x18160ddd;
+    // balanceOf(address account)
+    public static final int ABI_ID_ERC_BALANCE_OF_TOKEN = 0x70a08231;
+    // transfer(address recipient, uint256 amount)
+    public static final int ABI_ID_ERC_TRANSFER = 0xa9059cbb;
+    // transferFrom(address sender, address recipient, uint256 amount)
+    // transferFrom(address from, address to, uint256 tokenId)
+    public static final int ABI_ID_ERC_TRANSFER_FROM = 0x23b872dd;
+    // allowance(address owner, address spender)
+    public static final int ABI_ID_ERC_ALLOWANCE = 0xdd62ed3e;
+    // approve(address spender, uint256 amount)
+    // approve(address to, uint256 tokenId)
+    public static final int ABI_ID_ERC_APPROVE = 0x95ea7b3;
+    // setApprovalForAll(address operator, bool approved)
+    public static final int ABI_ID_ERC_SET_APPROVAL_FOR_ALL = 0xa22cb465;
+    // getApproved(uint256 tokenId)
+    public static final int ABI_ID_ERC_GET_APPROVED = 0x081812fc;
+    // isApprovedForAll(address owner, address operator)
+    public static final int ABI_ID_ERC_IS_APPROVED_FOR_ALL = 0xe985e9c5;
+    // ownerOf(uint256 tokenId)
+    public static final int ABI_ID_ERC_OWNER_OF_NFT = 0x6352211e;
+    // tokenURI(uint256 tokenId)
+    public static final int ABI_ID_ERC_TOKEN_URI_NFT = 0xc87b56dd;
+    // wipeTokenAccount(address, address, uint32)
+    public static final int ABI_WIPE_TOKEN_ACCOUNT_FUNGIBLE = 0x9790686d;
+    // wipeTokenAccount(address, address, int64[])
```

Comment:
Update signature based on the new method name defined in HIP-514 -> wipeTokenAccountNFT